lib/rabbit.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const stathat = app.locals.stathat;
+const logger = app.locals.logger;
+
+let channel = {};
+
+function onError(err) {
+  logger.error('Rabbit Error', err);
+  stathat(`gambit-core rabbit error ${err.code}`);
+}
+
+function makeConnection() {
+  logger.info('Attempting amqp connection');
+
+  const amqp = require('amqplib');
+
+  amqp.connect(process.env.RABBIT_URI)
+  .then(connection => connection.createChannel())
+  .then(ch => {
```

Comment:
Yeah it was a bit hard to name this because I couldn't use `channel`. Suggestions?